### PR TITLE
E2E test and documentation

### PR DESCRIPTION
**Pull request by humaton**
Updated documentation
Added script to run e2e tests
Changed script formatting

### CODE REVIEW

File: openshift/README.md [hunk 1]
```diff
@@ -30,3 +31,14 @@ CentosCI will build your image and comment on the PR with an image name and tag.
 Your image is available in the registry: docker pull registry.devshift.net/fabric8-analytics/worker-scaler:SNAPSHOT-PR-25
 
 To update your dev deployment use [oc edit](https://docs.openshift.com/container-platform/3.4/cli_reference/basic_cli_operations.html#edit) from the command line or editor in web interface.
+
+## Run E2E test agaist your deployment
```
Comment (msrb) [inline openshift/README.md]: I am missing information on how to actually run them :) I don't expect people to read the whole README file in integration-tests/ directory. This should be straightforward, i.e. do this-this-this and bum! the tests are running.

Comment (humaton) [reply openshift/README.md]: Indeed I have marked this as WIP. Updating it continuously...

File: openshift/run-2e2-test.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash -e
+here=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+F8A_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_JOB_API_URL=$(oc get route bayesian-jobs | awk '{print$2}' | grep -w "${OC_USERNAME}")
+F8A_ANITYA_API_URL=$(oc get route bayesian-api | awk '{print$2}' | grep -w "${OC_USERNAME}")
+
+#install python dependencies
+pip install --user -r $here/../integration-tests/requirements.txt
```
Comment (msrb) [inline openshift/run-2e2-test.sh]: Do we need this here?

https://github.com/fabric8-analytics/fabric8-analytics-common/blob/master/integration-tests/runtest.sh#L9

File: openshift/run-2e2-test.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+#check for configuration
+if [ "${F8A_API_URL}" == "Not set" ]; then
```
Comment (msrb) [inline openshift/run-2e2-test.sh]: Where do we set these variables to "Not set"?

File: openshift/README.md [hunk 4]
```diff
@@ -4,6 +4,11 @@
 
 Use your preferred package manager to install awk, aws-cli, psql, origin-clients, pwgen
 
+on Fedora:
+`sudo dnf install awk aws-tools aws-shell pwgen postgresql origin-clients`
```
Comment (msrb) [inline openshift/README.md]: What's in `aws-tools` and `aws-shell`? I have:
```
$ rpm -qf `which aws`
awscli-1.11.133-1.fc26.noarch

```

File: openshift/README.md [hunk 5]
```diff
@@ -30,3 +42,10 @@ CentosCI will build your image and comment on the PR with an image name and tag.
 Your image is available in the registry: docker pull registry.devshift.net/fabric8-analytics/worker-scaler:SNAPSHOT-PR-25
 
 To update your dev deployment use [oc edit](https://docs.openshift.com/container-platform/3.4/cli_reference/basic_cli_operations.html#edit) from the command line or editor in web interface.
+
+## Run E2E test agaist your deployment
+
+Environment variables are set by running deploy.sh.
```
Comment (msrb) [inline openshift/README.md]: Wouldn't be more fool-proof approach here to directly source `env.sh` inside `run-2e2-test.sh`?

i.e.:
```
. env.sh
```

File: openshift/run-2e2-test.sh (hunk 3 above)
Comment (humaton) [reply openshift/run-2e2-test.sh]: https://github.com/fabric8-analytics/fabric8-analytics-common/pull/394/commits/ba5a4fcd29f9275978c22d3b78a8a95fb64a650a

File: openshift/README.md [hunk 6]
```diff
@@ -4,6 +4,11 @@
 
 Use your preferred package manager to install awk, aws-cli, psql, origin-clients, pwgen
 
+on Fedora:
+`sudo dnf install awk awscli pwgen postgresql origin-clients`
```
Comment (jpopelka) [inline openshift/README.md]: on Fedora it's `gawk` too:

```
# dnf install awk
No match for argument: awk
Error: Unable to find a match

# dnf install gawk
Package gawk-4.1.4-6.fc26.x86_64 is already installed, skipping.
```

Comment (jpopelka) [reply openshift/README.md]: addressed in #397

**Pull request merged**